CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 2.0.4
+### ✨ Features and improvements
+- Changed workflow to use common release file from [#217](https://github.com/aws-geospatial/amazon-location-for-maplibre-gl-geocoder/pull/217), which would automate the github and node release process.
```

Comment:
Please update this to be focused on improvements for the customer (e.g. in this case, just dependency version updates + fixed incorrect map style example in README)